src/5e-SRD-StartingEquipment.json

```diff
@@ -4,844 +4,678 @@
-			"item": { "url": "/api/equipment/dagger", "name": "Dagger"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/greatclub", "name": "Greatclub"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/light-hammer", "name": "Light hammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/mace", "name": "Mace"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/quarterstaff", "name": "Quarterstaff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sickle", "name": "Sickle"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/spear", "name": "Spear"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dart", "name": "Dart"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortbow", "name": "Shortbow"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sling", "name": "Sling"}, "quantity": 1}]
-	}],
-	"choice_2": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/component-pouch", "name": "Component pouch"}, "quantity": 2}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/crystal", "name": "Crystal"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/orb", "name": "Orb"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/rod", "name": "Rod"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/staff", "name": "Staff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/wand", "name": "Wand"}, "quantity": 1}]
-	}],
-	"choice_3": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/dungeoneers-pack", "name": "Dungeoneer's Pack"}, "quantity": 1}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/explorers-pack", "name": "Explorer's Pack"}, "quantity": 1}]
-	}],
+	"starting_equipment": [
+		{ "equipment": { "url": "/api/equipment/dagger", "name": "Dagger" }, "quantity": 2 }
+	],
+	"starting_equipment_options": [
+		{
+			"choose": 1,
+			"type": "equipment",
+			"from": [
+				[
+					{ "equipment": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light" }, "quantity": 1 },
+					{ "equipment": { "url": "/api/equipment/crossbow-bolt", "name": "Crossbow bolt" }, "quantity": 20 }
+				], {
+					"equipment_option": {
+						"choose": 1,
+						"type": "equipment",
+						"from": {
+							"equipment_category": {
+								"name": "Simple Weapons",
+								"url": "/api/equipment-categories/simple-weapons"
+							}
+						}
+					}
+				}
+			]
+		}, {
+			"choose": 1,
+			"type": "equipment",
+			"from": [
+				{ "equipment": { "url": "/api/equipment/component-pouch", "name": "Component pouch" }, "quantity": 2 },
```

Comment:
Is the quantity supposed to be 2? I think this might be repeated in a few other places as well.

Reply:
This must be an artefact from the old data. Will update it.